computer_vision/flip_augmentation.py

```diff
@@ -0,0 +1,134 @@
+import glob
+import os
+import random
+from string import ascii_lowercase, digits
+
+import cv2
+
+
+"""
+Flip image and bounding box for computer vision task
+https://paperswithcode.com/method/randomhorizontalflip
+"""
+
+# Params
+LABEL_DIR = ""
+IMAGE_DIR = ""
+OUTPUT_DIR = ""
+FLIP_TYPE = 1  # (0 is vertical, 1 is horizontal)
+
+
+def main() -> None:
+    """
+    Get images list and annotations list from input dir.
+    Update new images and annotations.
+    Save images and annotations in output dir.
+    >>> pass  # A doctest is not possible for this function.
+    """
+    img_paths, annos = get_dataset(LABEL_DIR, IMAGE_DIR)
+    print("Processing...")
+    new_image, new_annos, path = update_image_and_anno(img_paths, annos, FLIP_TYPE)
+
+    for index in range(len(new_image)):
```

Comment:
`range(len())` is almost always a sign to use `enumerate()` instead...
```suggestion
    for index, image in enumerate(new_images):
```
And then below, replace `new_images[index]` with `image`.

Even slicker would be:
```
    for image, path in zip(new_images, paths):
    # or if you still really need index...
    for index, image, path in enumerate(zip(new_images, paths)):